src/tests/unittests/qiskit/test_passes.py

```diff
@@ -0,0 +1,180 @@
+import unittest
+
+import numpy as np
+import qiskit.quantum_info as qi
+from qiskit.circuit import QuantumCircuit
+from qiskit.circuit.library import CZGate, HGate, RXGate, RYGate, RZGate
+
+from qtt.qiskit.passes import (DecomposeCX, DecomposeU, DelayPass,
+                               LinearTopologyParallelPass,
+                               RemoveDiagonalGatesAfterInput,
+                               RemoveSmallRotations, SequentialPass)
+
+
+class TestQiskitPasses(unittest.TestCase):
```

Comment:
The tests are not testing edge cases of all passes. From code review I cannot tell if the passes are error free in edge cases.